input/docs/handbook/collections/index.md

```diff
@@ -82,14 +82,123 @@ var myBindingOperation = mySource
-an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update.
+If you have a unique id, you should use an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update. `SourceCache` has several performance advantages over `SourceList`, so if possible, always prefer `SourceCache` over `SourceList`.
 
 There is another difference. The cache side of dynamic data is much more mature and has a wider range of operators. Having more operators is mainly because I found it easier to achieve good all round performance with the key based operators and do not want to add anything to Dynamic Data which inherently has poor performance.
 
-## How to track changes in collections of Reactive Objects?
+# Using DynamicData with ReactiveUI
+
+When building applications with ReactiveUI and DynamicData, you have a choice to work with mutable or with immutable collections. When working with immutable ones, using an `ObservableAsPropertyHelper<T>` is enough in simple cases. The `ObservableAsPropertyHelper<T>` represents an `Observable<T>`, a stream of values over time. You can treat those values as events, and the new values as event arguments. This means if you are using immutable collections, you can treat them as event arguments and update a property with a new collection each time it changes. See [ObservableAsPropertyHelper Handbook section](/docs/handbook/oaph/) to learn how to use this feature. Note, that creating a new colletion for each update degrades performance and should be generally avoided, prefer to use DynamicData instead.
+
+### An Example
+
+Imagine your application needs a service that will expose a collection mutated by a background worker. You need to get change notifications from it somehow to synchronize it with the user interface. Here DynamicData comes to the rescue. You expose an `IObservable<IChangeSet<bool>>` from your service to the outer world, and DynamicData takes case of allowing you to observe changes of your mutable `SourceList` of items. Use the `.Connect()` operator to turn your `SourceList<T>` to an observable change set `IObservable<IChangeSet<bool>>`.
+
+```cs
+public class Service 
+{
+    private readonly SourceList<bool> _items = new SourceList<bool>();
+
+    public IObservable<IChangeSet<bool>> Items { get; }
+
+    public Service()
+    {
+        Items = _items
+            // Use the .Connect() operator to connect to the 
+            // data set and turn it into a DynamicData monad.
+            // This gives you the ability to use all DynamicData
+            // LINQ-like query operators. 
+            .Connect()
+            // Publish() creates a Subject<T> after the first 
```

Comment:
The is not the correct use of `Publish`.  `Publish`  returns `IConnectableObservable()` which should only be used when there are a known consumers.  `Items = _items .Connect()` is sufficient.